ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimMapper.java

```diff
@@ -37,7 +38,8 @@ public ClaimMapper(PersonalInjuryMapper personalInjuryMapper,
                        AmountMapper amountMapper,
                        PaymentMapper paymentMapper,
                        InterestMapper interestMapper,
-                       InterestDateMapper interestDateMapper) {
+                       InterestDateMapper interestDateMapper,
+                       TimelineMapper timelineMapper) {
```

Comment:
move `) {` to a new line to reduce future diffs?

Reply:
done